lib/reverse_sentence.rb

```diff
@@ -1,4 +1,48 @@
 # A method to reverse the words in a sentence, in place.
+# Time Complexity: O(3n) * O(n) + O(n) = O(n^2) + O(n) = O(n^2)
+# Space Complexity: O(1)
+
 def reverse_sentence(my_sentence)
```

Comment:
As a further enhancement, you could consider making your code modular and reusable by having separate methods created for reversing words and for partially reversing a string from a start index to an end index.